### ISSUE TITLE
Prediction for new users

### ISSUE THREAD

**Issue by neeravmakwana** [NicolasHug/Surprise#113]
#### Description

This is not an issue but a query and could be slightly similar to [this query](https://github.com/NicolasHug/Surprise/issues/103).

I wish to train the model once and based on what a new user inputs (only the user is new, items are the same as in test set), provide recommendations.

Is there any way I by which I can predict for the new user without having to incorporate this new data into the training set and retraining the model? I am currently using SVD model, but am open to using any model.

My training set has more than 2.5 million rows and it takes hours to train the model, so I want to avoid retraining the model as far as possible.

For example, the new user will input 5 items (which are already there in the training set) and all need to be considered as a rating of 5. I then need to recommend top N items on the basis of these items.

Can this be done without retraining? Thanks!

**Comment by NicolasHug** [comment]
Hi,
No, as mentioned in #103 and #15, the only way is currently to train on the augmented training set.

I plan to address incremental learning but I don't know when it will be done. Although I think that for SVD, incremental learning basically reduces to a regular training step. The only difference is that the biases and factors are already initialized form the previous training step. (see #15). This should be fairly easy to implement.

**Comment by neeravmakwana** [comment]
Thanks Nicolas, I'll try figuring that out.
